virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc

```diff
@@ -6,13 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can connect a virtual machine (VM) to an Open Virtual Network (OVN)-Kubernetes secondary network. {VirtProductName} supports the layer 2 and localnet topologies for OVN-Kubernetes.
+ifdef::openshift-rosa,openshift-dedicated[]
+You can connect a VM to an Open Virtual Network (OVN)-Kubernetes secondary network. {VirtProductName} supports the `layer2` topology for OVN-Kubernetes. 
+endif::openshift-rosa,openshift-dedicated[]
+
+ifndef::openshift-rosa,openshift-dedicated[]
+You can connect a virtual machine (VM) to an OVN-Kubernetes secondary network. {VirtProductName} supports the `layer2` and `localnet` topologies for OVN-Kubernetes.
 
-* A layer 2 topology connects workloads by a cluster-wide logical switch. The OVN-Kubernetes Container Network Interface (CNI) plug-in uses the Geneve (Generic Network Virtualization Encapsulation) protocol to create an overlay network between nodes. You can use this overlay network to connect VMs on different nodes, without having to configure any additional physical networking infrastructure.
+* A `layer2` topology connects workloads by a cluster-wide logical switch. The OVN-Kubernetes Container Network Interface (CNI) plug-in uses the Geneve (Generic Network Virtualization Encapsulation) protocol to create an overlay network between nodes. You can use this overlay network to connect VMs on different nodes, without having to configure any additional physical networking infrastructure.
```

Comment:
We discussed it offline with Shikha, but I'll repeat it here for the sake of transparency.

Using `layer2` everywhere can look odd sometimes. On the other hand, "layer 2" alone is overused and could mislead people, as Miguel commented already.

I think it's ok to go with a compromise - spelling it with backtick first, but then use with backticks or without, based on your taste / judgement / context.

FWIW, the mix of the two currently used in the document looks good to me.